algolia/algoliasearch-sfcc-b2c
# fix: add node 14/16 support

`npm install` currently only works with Node.js v10
This was because of outdated WebdriverIO packages. Bumping their version fixed the issue.

But after analyzing all the dependencies, it seems that most of them were carried away from other projects and not used at all, so I've removed the majority of them, and bumped the versions of the remaining ones.

`npm install` now works with Node.js versions 14 and 16.

### How to test

I've tested that the NPM commands still work as expected:
- `npm test`
- `npm run lint`
- `npm run uploadCartridge`
